sklearn/gaussian_process/tests/test_gaussian_process.py

```diff
@@ -15,6 +15,7 @@
 from sklearn.datasets import make_regression
 from sklearn.utils.testing import assert_greater, assert_true, assert_raises
 
+pytestmark = pytest.mark.filterwarnings('ignore', category=DeprecationWarning)
```

Comment:
> What is the reason that DeprecationWarnings are ignored here? (this is not part of the common tests)

Unless I am missing something sklearn/gaussian_process.py is deprecated and scheduled to be removed in 0.20.

Reply:
Ah, that's a good reason :-) 
Ideally I would say that deprecation warnings are asserted once and then explicitly ignored for other tests, but since this is deprecating a full sub-module and not a method or so, then probably makes sense to just ignore them for the full submodule.